### DISCUSSION TITLE
Prevent Closing when hovered

### DISCUSSION THREAD

**Discussion by coolsam726** [emmanuelsw/notiwind#6] [Ideas]
First of all, thanks for extending the package to support transitions and fixing the vue 3 + vite issues.

This is a **feature request**, to have the notification staying active (prevent closing) when the user hovers on the notification. This will allow one to examine the message/error carefully in case they need more time to investigate it.

**Comment by coolsam726** [comment]
@emmanuelsw I have an idea of a PR I can do to support this, but I would like to hear your thoughts on the same first.

**Comment by emmanuelsw** [comment]
Hey! you're welcome.
I think it would be nice to add that functionality with the possibility to choose whether to enable it or not. Many of the notification libraries provide the same functionality so I wouldn't have a problem with this one doing it too.

**Comment by Joao-VictorF** [comment]
Hey, I needed this functionality in a project and found this issue, so I created a PR to solve it =) https://github.com/emmanuelsw/notiwind/pull/38

**Comment by emmanuelsw** [comment]
Added in v2.0.2.
